recipe/meta.yaml

```diff
@@ -45,12 +45,16 @@ test:
     - pip
     - netcdf4
     - geocat-datafiles
+    - numba
```

Comment:
We aren't actually using numba in geocat-comp (yet), I don't think we should add it to the `meta.yml` here.

Reply:
Note for future developers: numba is needed by xskillscore